tests/unit/PackagesTest.php

```diff
@@ -37,15 +37,4 @@ public function test_returns_count(): void
             new Package(new PackageName('a/b'), []),
         ]));
     }
-
-    public function test_can_be_iterated(): void
```

Comment:
feel like this test was useless anyway

Reply:
agree - although couldn't it have been rewritten to be more like this? 🤔 

```
$packages = new Packages([
    new Package(new PackageName('a/a'), []),
    new Package(new PackageName('a/b'), []),
]);

self::assertEquals(new Package(new PackageName('a/a'), []), $packages->getIterator()->current());
self::assertEquals(new Package(new PackageName('b/b'), []), $packages->getIterator()->next()->current());
```

Reply:
oh, wait, we already have `test_returns_instance_with_sorted_packages`